props/saber_fett263_buttons.h

```diff
@@ -1918,18 +1918,23 @@ SaberFett263Buttons() : PropBase() {}
       } else {
         push_begin_millis_ = millis();
       }
-
     } else {
+      if (!fusor.ready()) {
+          fusor_ready_millis_ = millis();
+      }
       // EVENT_SWING - Swing On gesture control to allow fine tuning of speed needed to ignite
       if (menu_ || millis() - saber_off_time_millis_ < MOTION_TIMEOUT) {
         SaberBase::RequestMotion();
         if (swinging_ && fusor.swing_speed() < 90) {
           swinging_ = false;
         }
-        if (!swinging_ && fusor.swing_speed() > saved_gesture_control.swingonspeed) {
-          swinging_ = true;
-          Event(BUTTON_NONE, EVENT_SWING);
-        }
+        // Prevent false swing during motion startup
+        if (millis() - fusor_ready_millis_ > 2000) {
```

Comment:
if (A) {
    if (B) {
    }
}

should generally be if (A && B) { }

Reply:
fixed